base_layer/wallet_ffi/src/callback_handler.rs

```diff
@@ -265,181 +255,181 @@ where TBackend: TransactionBackend + 'static
+                                                base_node_state.best_block_height = final_height;
+                                                base_node_state.latency = u64::try_from(latency.as_millis()).unwrap_or(u64::MAX);
+                                                self.base_node_state_changed(base_node_state);
+                                            },
+                                            _ => {}
+                                        }
+                                    },
+                                    Err(e) => {
+                                        error!(target: LOG_TARGET, "Problem with utxo scanner: {}",e);
+                                    },
+                            },
+
+                            result = self.dht_event_stream.recv() => {
+                                match result {
+                                    Ok(msg) => {
+                                        trace!(target: LOG_TARGET, "DHT Callback Handler event {:?}", msg);
+                                        if let DhtEvent::StoreAndForwardMessagesReceived = *msg {
+                                            self.saf_messages_received_event();
+                                        }
+                                    },
+                                    Err(_e) => error!(target: LOG_TARGET, "Error reading from DHT event broadcast channel"),
                                 }
-                                UtxoScannerEvent::Completed {
-                                    final_height,
-                                    ..
-                                }=> {
-                                self.scanned_height_changed(final_height);
-                                },
-                                _ => {}
-                            }
-                        },
-                        Err(e) => {
-                            error!(target: LOG_TARGET, "Problem with utxo scanner: {}",e);
-                        },
-                },
-
-                result = self.dht_event_stream.recv() => {
-                    match result {
-                        Ok(msg) => {
-                            trace!(target: LOG_TARGET, "DHT Callback Handler event {:?}", msg);
-                            if let DhtEvent::StoreAndForwardMessagesReceived = *msg {
-                                self.saf_messages_received_event();
-                            }
-                        },
-                        Err(_e) => error!(target: LOG_TARGET, "Error reading from DHT event broadcast channel"),
-                    }
-                }
-
-                Ok(_) = self.connectivity_status_watch.changed() => {
-                    let status  = *self.connectivity_status_watch.borrow();
-                    trace!(target: LOG_TARGET, "Connectivity status change detected: {:?}", status);
-                    self.connectivity_status_changed(status);
-                },
```

Comment:
I think we should still emit events for connectivity status changed - the mobile wallet needs to know if it is offline - once the status changes from offline to online it can also change back to offline again.

Reply:
we dont know if are offline